android/app/build.gradle

```diff
@@ -129,7 +129,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "com.ihsanreactnative"
+        applicationId "com.ihsanproject.client"
```

Comment:
Why do we have to change this?

Reply:
The Google SSO Firebase project (which I already created for ihsanproject) was named that. They need to match. For khatmah we'll create a new firebase project and align the names when we clone this project

Reply:
Okay, makes sense. It doesn't mess with the build of the react project right? It was really delicate trying to rename the project from `ihsanReactNative` to `IhsanReactNative` just because of all those super obscure files